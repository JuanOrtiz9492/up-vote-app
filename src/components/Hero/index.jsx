import React from 'react';
import styled from 'styled-components';

const StyledContainer = styled.div`
    display: grid;
    grid-template-rows: 25% 65% 10%;
    background-image: url(${props => require(`../../assets/img/${props.image}`)});
    width: 1200px;
    height: 630px;
`

const Hero = ({}) => (
    <StyledContainer image="PopeFrancis.jpg">
        <div>uno</div>
        <div>dois</div>
        <div>tres</div>
    </StyledContainer>
)

export default Hero;