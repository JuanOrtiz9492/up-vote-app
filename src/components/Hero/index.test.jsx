import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from "react-router-dom";
import Hero from '.';

describe('Hero Component', () => {

    it('should Match the Snapshot', () => {
        const { container } = render(<Router><Hero /></Router>);
        expect(container).toMatchSnapshot();
      })
})

