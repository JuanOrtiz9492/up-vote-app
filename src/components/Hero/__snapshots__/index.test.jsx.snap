// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Component should Match the Snapshot 1`] = `
<div>
  <div
    class="sc-fzXfLW gHRLEZ"
  >
    <header
      class="sc-AykKI fgxbaB"
    >
      <h1
        class="sc-AykKK RakPd"
      >
        Rule of Thumb.
      </h1>
      <nav
        class="sc-AykKJ chLIen"
      >
        <a
          class="sc-fzXfLO kyFEms"
          href="/past-trials"
        >
          Past Trials
        </a>
        <a
          class="sc-fzXfLO kyFEms"
          href="/"
        >
          How it Works
        </a>
        <a
          class="sc-fzXfLO kyFEms"
          href="/"
        >
          Log in/Sign Up
        </a>
        <a
          class="sc-fzXfLO kyFEms"
          href="/"
        >
          <svg
            class="sc-AykKE diYqms"
            color="#ffffff"
            width="2"
          >
            search.svg
          </svg>
        </a>
      </nav>
    </header>
    <section
      class="sc-fzXfLQ faPqQk"
    >
      <div
        class="sc-fzXfLS hzfJPM"
      >
        <span
          class="sc-fzXfLR foYWHE"
        >
          whats's your opinion on
        </span>
        <span
          class="sc-fzXfLR jeomeD"
        >
          Pope Francis?
        </span>
        <span
          class="sc-fzXfLR hjdUXp"
        >
          He's talking tought on clergy sexual abuse, but is he just another papal pervert protector? (thumbs down) or a true pedophile punishing pontiff? (thumbs up)
        </span>
        <a
          class="sc-fzXfLX ldyza-d"
          href="#"
        >
          <svg
            class="sc-AykKF cPgGiI"
            color="#ffffff"
            width="1"
          >
            wikipedia-w.svg
          </svg>
           More Information
        </a>
        <span
          class="sc-fzXfLR fOtazo"
        >
          What's Your Veredict?
        </span>
      </div>
      <div>
        <button
          class="sc-fzXfLP iHWoqw jungle"
          color="#ffffff"
          height="100%"
          width="50%"
        >
          <svg
            class="sc-AykKC jytkdL"
            color="#ffffff"
            width="3"
          >
            thumbs-up.svg
          </svg>
        </button>
        <button
          class="sc-fzXfLP iHWoqw fire"
          color="#ffffff"
          height="100%"
          width="50%"
        >
          <svg
            class="sc-AykKD dCQaqc"
            color="#ffffff"
            width="3"
          >
            thumbs-down.svg
          </svg>
        </button>
      </div>
    </section>
    <div
      class="sc-fzXfLT kFnChX"
    >
      <div
        class="sc-fzXfLU bnOoIG"
        value="37.142857142857146"
      >
        <span
          class="sc-fzXfLV fUPZAy leftText"
          color="#ffffff"
          font-size="0.8em"
        >
          CLOSING IN
        </span>
      </div>
      <span
        class="sc-fzXfLV iStdJW rightText"
        color="#333333"
        font-size="1.5em"
      >
        22
         days
      </span>
    </div>
  </div>
</div>
`;
